backend/cellguide/pipeline/utils.py

```diff
@@ -19,8 +19,7 @@ def output_json(data, path):
     path - str
     """
 
-    if is_dataclass(data):
```

Comment:
This was wrong - data can be a dictionary of dataclasses, which is itself not a dataclass